lib/workload/stateful/filemanager/deploy/stack/filemanager_stack.ts

```diff
@@ -100,6 +102,52 @@ export class FilemanagerStack extends Stack {
     const eventSource = new lambdaEventSources.SqsEventSource(queue);
     filemanagerLambda.addEventSource(eventSource);
 
-    // todo RDS instance.
+    // VPC
+    //const vpc = ec2.Vpc.fromLookup(this, 'main-vpc', { isDefault: false });
+    const vpc = new ec2.Vpc(this, 'vpc', {
+      ipAddresses: ec2.IpAddresses.cidr('10.0.0.0/24'),
+      maxAzs: 99, // As many as there are available in the region
+      natGateways: 1,
+      subnetConfiguration: [
+        {
+          name: 'ingress',
+          subnetType: ec2.SubnetType.PUBLIC,
+        },
+        {
+          name: 'application',
+          subnetType: ec2.SubnetType.PRIVATE_WITH_EGRESS,
+        },
+        {
+          name: 'database',
+          subnetType: ec2.SubnetType.PRIVATE_ISOLATED,
+        },
+      ],
+    });
+
+    // Secret
+    new secretsmanager.Secret(this, 'filemanager_db_secret', {
+      secretName: 'filemanager_db_secret', // pragma: allowlist secret
+      generateSecretString: {
+        secretStringTemplate: JSON.stringify({ username: 'filemanager' }),
+        excludePunctuation: true,
+        generateStringKey: 'password',
+      },
+    });
+
+    // RDS
+    new rds.DatabaseCluster(this, 'Database', {
+      engine: rds.DatabaseClusterEngine.auroraPostgres({
+        version: rds.AuroraPostgresEngineVersion.VER_15_3,
```

Comment:
Do we want to use postgres 16? Or does Aurora not support it yet?

Reply:
It was not on the enum/autocomplete 🤷🏻

Reply:
It looks like it is supported, although only very recently (from November 15), so it's probably not in CDK yet. Should be able to set it manually though: https://github.com/aws/aws-cdk/issues/23253#issuecomment-1340935712.